crud.py

```diff
@@ -0,0 +1,31 @@
+from sqlalchemy.orm import Session
+
+from db.models import DBBook, DBAuthor
+import schemas
+
+
+def create_author(db: Session, author: schemas.AuthorCreate):
+    db_author = DBAuthor(**author.dict())
+    db.add(db_author)
+    db.commit()
+    db.refresh(db_author)
+    return db_author
+
+def get_authors(db: Session, skip: int = 0, limit: int = 10):
+    return db.query(DBAuthor).offset(skip).limit(limit).all()
+
+def get_author_by_id(db: Session, author_id: int):
+    return db.query(DBAuthor).filter(DBAuthor.id == author_id).first()
+
+def create_book(db: Session, author: schemas.BookCreate):
```

Comment:
The parameter `author` in the `create_book` function should likely be `book` to match the usage in line 21. This seems to be a typo, as you are trying to create a `DBBook` instance.